### DISCUSSION TITLE
General DiscussionWhy can't I see the knowledge base created by others, even though I am an admin and owner? The context is that my colleague has left the company, but I can no longer see the knowledge base they created.

### DISCUSSION THREAD

**Discussion by lianhj** [langgenius/dify#8518] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Why can't I see the knowledge base created by others, even though I am an admin and owner? The context is that my colleague has left the company, but I can no longer see the knowledge base they created.

**Comment by mamazary** [comment]
Update, The problem is in knowledge based / dataset permission. When we create a new dataset from web/FE, we don't have any setting regarding dataset permission, only dataset name. So it would automatically be a private dataset, if we want to share with the members in the workspace, we need to create using API and pass permission to all_team_members or partial_members.

For existing dataset, you can go to setting and change the permission.
